svg-radial.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+angular.module( "svgRadial", [] )
+    .directive( "radialProgress", function ()
+    {
+        return {
+            restrict: "E",
```

Comment:
There's no particular reason for this yet. It was just how it was included in our project at the time. All of this is mostly copy/paste.